readme/configuration-management.md

```diff
@@ -70,14 +70,39 @@ We need to find a better way of preventing this than manually monitoring module
 
 TODO: Update this documentation once Config Split's greylist functionality has been better documented and tested.
 
 The second limitation is that it's difficult to define configuration that varies between sites in a multisite installation. Multisite installations that require highly customized bundles of configuration per-site might be better suited by a Features-based workflow.
 
+### Setting up Config Split
+
+This section will guide you through initially setting up a Config Split workflow on your BLT based project.
+
+1. Ensure that you are using Drush version 8.1.10 or higher.
+2. Add Config Split version 8.x-1.0-beta4 or higher to your codebase: `composer require drupal/config_split`
+3. TODO: finish this section once the related functionality is stabilized in BLT core.
+
+### Using Config Split to capture and deploy configuration
+
+BLT and Config Split together make it easy to capture configuration changes in code and deploy those changes to a remote environment.
+
+As a prerequisite, make sure your BLT-based project is set up to use Config Split (see section above).
+
+To capture and deploy configuration changes using Config Split:
+
+1. Ensure that your local environment is up to date and refreshed (e.g. `git pull` and `blt local:refresh`).
+2. Use the Drupal UI to make the necessary configuration changes in your local environment. For instance, go to http://local.example.com/admin/structure/types/add to add a new content type.
+3. Once you have completed local development, use `drush cex` (`config-export`) to export your configuration changes to the `config/default` directory. Remember to use an appropriate alias if you are using a VM (e.g. `drush @example.local cex`).
+4. Review the updated configuration in `config/default` using `git diff`.  If you are satisfied with the changes, commit them and open a pull request.
+
+Note that when you run `drush cex`, if the project has been configured correctly, some configuration that's specific to the development environment should automatically be excluded. Similarly, some configuration that's intended to be "unlocked" in production might also be excluded (such as webforms). If you need to customize this behavior, you can modify the blacklists or greylists by following the steps in the section above.
+
 ## Features-based workflow
 
 Features allows you to bundle related configuration files (such as a content type and its fields) into individual feature modules. Drupal treats features just like normal modules, but Features and its dependencies add some special sauce that allow features to not only provide default configuration (like normal modules), but to also update (track and import) changes to this configuration.
```

Comment:
We need to indicate that `cm.strategy` must be set to `features` in project.yml.